src/main/java/lotto/util/GameRule.java

```diff
@@ -0,0 +1,22 @@
+package lotto.util;
+
+public class GameRule {
+
+    private GameRule() {
+
+    }
+
+    public static final int LOTTO_PAPER_PRICE = 1000;
+    public static final int LOTTO_START_NUMBER = 1;
+    public static final int LOTTO_END_NUMBER = 45;
+    public static final int LOTTO_START_INDEX = 0;
+    public static final int LOTTO_END_INDEX = 6;
+
+    public static final int MATCH_PRICE_3 = 5000;
+    public static final int MATCH_PRICE_4 = 50000;
+    public static final int MATCH_PRICE_5 = 1500000;
+    public static final int MATCH_PRICE_6 = 2000000000;
```

Comment:
숫자에 `언더 스코어`를 활용하여 가독성을 높힐 수 있습니다.

```suggestion
    public static final int MATCH_PRICE_6 = 2_000_000_000;
```

Reply:
언더 스코어에 대해서 배워갑니다.